colcon_core/subprocess.py

```diff
@@ -49,7 +49,8 @@ async def run(
     cwd: str = None,
     env: Mapping[str, str] = None,
     shell: bool = False,
-    use_pty: Optional[bool] = None
+    use_pty: Optional[bool] = None,
+    close_fds: Optional[bool] = None
```

Comment:
In the underlying Python API the default value of this keyword argument is `True`. Is there a reason the default value here diverges from that?

Reply:
It isn't `True` on Windows prior to Python 3.7. In fact, it can't be `True` if the standard streams are redirected on Windows prior to 3.7.